zipline/data/us_equity_minutes.py

```diff
@@ -49,10 +48,29 @@ def write_metadata(directory, first_trading_day):
         json.dump(metadata, fp)
 
 
+def _bcolz_minute_index(trading_days):
+        minutes = np.zeros(len(trading_days) * MINUTES_PER_DAY,
+                           dtype='datetime64[ns]')
+        market_opens = tradingcalendar.open_and_closes.market_open
+        mask = market_opens.index.slice_indexer(start=trading_days[0],
+                                                end=trading_days[-1])
+        opens = market_opens[mask]
+        for i, market_open in enumerate(opens):
+            start = market_open.value
+            end = start + _NANOS_IN_MINUTE * MINUTES_PER_DAY
+            minute_values = np.arange(start, end, _NANOS_IN_MINUTE)
+            start_ix = MINUTES_PER_DAY * i
+            end_ix = start_ix + MINUTES_PER_DAY
+            minutes[start_ix:end_ix] = minute_values
+        return pd.to_datetime(minutes, utc=True, box=True)
+
+
 class BcolzMinuteBarWriter(with_metaclass(ABCMeta)):
     """
     Class capable of writing minute OHLCV data to disk into bcolz format.
     """
+    _writer_env = TradingEnvironment()
```

Comment:
This was previously at module scope. The change to be a class member was in effort to ease this class into using passing the env directly, since the value was needed in `_write_internal` with the change to how the minutes were created during the asset loop.

Shall we make this an instance member instead and pass it in with the init?